pkg/apis/networking/v1alpha1/service_types.go

```diff
@@ -30,6 +30,12 @@ const (
 	ResourcePluralMultiClusterService = "multiclusterservices"
 	// ResourceNamespaceScopedMultiClusterService indicates if MultiClusterService is NamespaceScoped.
 	ResourceNamespaceScopedMultiClusterService = true
+	// ServiceApplied indicates whether the service is propagated to member clusters
+	ServiceApplied string = "ServiceApplied"
+	// EndpointSliceCollected indicates whether the endpointslice is collected from provision clusters
+	EndpointSliceCollected string = "EndpointSliceCollected"
+	// EndpointSliceSynced indicates whether the EndpointSlice is synced to consumption clusters
+	EndpointSliceSynced string = "EndpointSliceSynced"
```

Comment:
Please remember to update this name once updated the controller name.
https://github.com/karmada-io/karmada/issues/4292#issuecomment-1831185955

```suggestion
	EndpointSliceSynced string = "EndpointSliceDispatched"
```